libs/@local/graph/postgres-store/src/store/postgres/crud.rs

```diff
@@ -74,14 +74,14 @@ where
     type ReadPaginatedStream =
         impl Stream<Item = Result<Self::QueryResult, Report<QueryError>>> + Send + Sync;
 
-    #[tracing::instrument(level = "info", skip(self, filter, sorting))]
+    #[tracing::instrument(level = "info", skip(self, filters, sorting))]
     #[expect(
         clippy::type_complexity,
         reason = "The complexity comes from the `instrument` macro"
     )]
     async fn read_paginated(
         &self,
-        filter: &Filter<'_, R>,
+        filters: &[Filter<'_, R>],
```

Comment:
couldn't this be relaxed even further to `impl IntoIterator<Item = Filter<'_, R>`?